dev-util/sourcetrail/sourcetrail-2019.4.102.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="A free and open-source cross-platform source explorer"
+DESCRIPTION="Sourcetrail helps you get productive on unfamiliar source code"
+HOMEPAGE="https://www.sourcetrail.com/"
+COPYRIGHT="2019 Andreas Stallinger and others"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/CoatiSoftware/Sourcetrail/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="74b0f11904e5faaead94f92319001d77e23d102c7370f63f0f83a624e46312fe"
+SOURCE_DIR="Sourcetrail-$portVersion"
+PATCHES="sourcetrail-$portVersion.patchset"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURE="x86"
+
+PROVIDES="
+	sourcetrail$secondaryArchSuffix = $portVersion
+ 	cmd:sourcetrail
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_date_time$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libQt5$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Designer$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Quick$secondaryArchSuffix
+	lib:libQt5Test$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboost_date_time$secondaryArchSuffix >= 1.67
+	devel:libboost_filesystem$secondaryArchSuffix >= 1.67
+	devel:libboost_program_options$secondaryArchSuffix >= 1.67
+	devel:libboost_system$secondaryArchSuffix >= 1.67
+	devel:libQt5$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Designer$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:getconf
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_BUILD_TYPE="Release" \
+	-DBOOST_ROOT=$portPackageLinksDir/devel~libboost_program_options/develop \
+	-DQt5_DIR=$portPackageLinksDir/devel~Qt/version/platform/compiler/lib/cmake/Qt5 \
```

Comment:
Does this path really exist? $portPackageLinksDir/devel~Qt/version/platform/compiler/lib/cmake/Qt5
it *looks* like it doesn't exist: please try to remove this parameter.

Reply:
Already mentioned in the beginning, the lines for boost and Qt5 are not needed